packages/runtime/container-runtime/src/dataStoreContexts.ts

```diff
@@ -81,9 +81,23 @@ export class DataStoreContexts implements Iterable<[string, FluidDataStoreContex
 	public delete(id: string): boolean {
 		this.deferredContexts.delete(id);
 		this.notBoundContexts.delete(id);
+
+		// Stash the context here in case it's requested in this session, we can log some details about it
+		const context = this._contexts.get(id);
```

Comment:
Maybe we should add another log if the context being deleted has already been realized? That should not happen and this be a clear indication that somethine unexpected happened.